js/src/edit-block/components/header.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	PostPublishButton,
+	PostSavedState,
+	EditorHistoryRedo,
+	EditorHistoryUndo,
+} from '@wordpress/editor';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * The header component.
+ *
+ * @return {React.ReactElement} The header component.
+ */
+const Header = () => {
+	const backURL = addQueryArgs( 'edit.php', {
+		post_type: 'genesis_custom_block',
+	} );
+
+	return (
+		<div className="flex items-center h-16 border-b border-gray-300 w-full">
+			<a
+				className="flex items-center justify-center h-full w-16 bg-gray-900"
+				href={ backURL }
+			>
+				<svg
```

Comment:
Not a blocker, but if you want to avoid inlining the WP logo here you could do:

```js
import { Icon, wordpress } from '@wordpress/icons';
```

And then:

```js
			<a
				className="flex items-center justify-center h-full w-16 bg-gray-900 text-white"
				href={ backURL }
				aria-label={ __( 'Go back to WordPress', 'genesis-custom-blocks' ) }
			>
				<Icon size="36" className="fill-current" icon={ wordpress } />
			</a>
```

Which should give you the same appearance without having to maintain the logo (on the off-chance it gets adjusted).

Reply:
Oh, good idea!

Reply:
Great idea, applied in https://github.com/studiopress/genesis-custom-blocks/pull/41/commits/f1c7e257107ed1ff397f01e43c1ba575488d9a9a